tasks/02_kubernetes_fundamentals/kubernetes_4/k8s/dev/slave_apis_services.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: decision-tree-model-api
+  labels:
+    app: decision-tree-model-api
+spec:
+  clusterIP: 10.96.1.1
+  type: LoadBalancer
+  ports:
+  - port: 5000
+    targetPort: 5000
```

Comment:
Nit: Nie jestem pewien co do tych portow, bo "port", to jesli dobrze rozumiem jest port, przez ktory serwis "kontaktuje" sie z podem, natomiast targetport, to port na ktorym dziala aplikacja wewnatrz portu. Nalezaloby pewnie doczytac, czy to napewno dobra praktyka. Jesli jestescie co do tego pewni to olejcie ten kom, bo tez widze ze wiele razy to powielacie :)

source: https://stackoverflow.com/questions/49981601/difference-between-targetport-and-port-in-kubernetes-service-definition